game.py

```diff
@@ -51,18 +52,27 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, known=True, **kwargs)
 
 
-def lookup_game_by_name_or_alias(name):
-    # Name may contain extra junk, e.g. "I'd play cs later, after food" would mean name="cs later, after food"
-    for game in games:
+def lookup_known_game_by_name_or_alias(name):
+    for game in get_known_games():
         if game.loosely_matches(name):
             return game
-    return Game(name=name) if name else None
+
+def lookup_game_by_name_or_alias(name):
+    # Name may contain extra junk, e.g. "I'd play cs later, after food" would mean name="cs later, after food"
+    game = lookup_known_game_by_name_or_alias(name)
+    return game if game else Game(name=name)
 
 
-def get_known_games(json_filename='known_games.json'):
+def read_games_json_dict(json_filename=os.path.join(os.path.dirname(__file__), 'known_games.json')):
```

Comment:
The term json_dict is confusing (json is a string, a dict is a dict). How about `read_games_dict` and `write_games_dict`

More than that, you should rarely try to encode the type of the param into the name of something - if we change from json storage to yaml or a proper db then none of these functions are correct any more.

On the subject of putting 'dict' in the name, I'm not a fan of that either but I'm inclined to keep it to differentiate between having a structure of Game object. The correct thing to do is to refactor it so you never work with the dicts directly and Games know how to serialize themselves (or something does) but that's a bigger change. For now I think clarifying the name will be fine.